File: Collective/Examples/mpi_hello.py
#MPI Hello World
from mpi4py import MPI

cw = MPI.COMM_WORLD
rank = cw.rank
size = cw.size
proc = MPI.Get_processor_name()



if rank == 0:
    print("{} MPI processes are now active".format(size))

print('Hello from node {} rank {} out of {} mpi processes'.format(proc, rank, size))

File: Collective/Examples/mpi_hello.R
suppressMessages(library(pbdMPI, quietly = TRUE))

init()
.comm.size <- comm.size()
.comm.rank <- comm.rank()

# MPI_Get_processor_name doesn't exisit in pbdMPI

if (.comm.rank == 0) {
    print(sprintf("%d MPI processes are active", .comm.size))
}

comm.print(sprintf("Hello from rank %d out of %d mpi process", .comm.rank, .comm.size), all.rank = TRUE)

finalize()
